README.md

```diff
@@ -2,62 +2,111 @@
+            - Headed mode will spawn a browser window in the screen and execute the tests
+            - Accepted parameters: `--headed`
+            - Sample execution:
 
-- run all the tests in headed mode (this will load multiple browsers):
+                `npm run test -- about --headed`
 
-    `npm run test:headed`
+                `npm run test about -- --headed`
 
-- create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
+        - Test execution - [debug mode](https://playwright.dev/docs/running-tests#debug-tests-with-the-playwright-inspector):
+            - This will spawn a browser window and Playwright Inspector window in the screen which will help on inspecting the whole test execution flow step by step
+            - Accepted parameters: `--debug`
+            - Sample execution:
 
+                `npm run test -- about --debug`
 
-## Running individual tests by category (headless mode)
+                `npm run test about -- --debug`
 
-- run about tests: `npm run test:about`
-- run av tests: `npm run test:av`
-- run books tests: `npm run test:books`
-- run collection tests: `npm run test:collection`
-- run details tests: `npm run test:details`
-- run home tests: `npm run test:home`
-- run login tests: `npm run test:login`
-- run music tests: `npm run test:music`
-- run search tests: `npm run test:search`
-- run profile tests: `npm run test:profile`
+        - Test execution - [UI mode](https://playwright.dev/docs/running-tests#debug-tests-in-ui-mode):
+            - This will spawn a browser window and Playwright Inspector window in the screen which will help on inspecting the whole test execution flow step by step
+            - Accepted parameters: `--ui`
+            - Sample execution:
 
+                `npm run test -- about --ui`
 
-## Running individual tests by category (headed mode)
+        - Test execution with [Trace viewer](https://playwright.dev/docs/trace-viewer) mode:
+            - Trace viewer will record the test run which will be included in the Playwright report
+            - It is disabled by default, you can enable it with the command below
+            - Accepted parameters: `trace`
+            - Sample execution:
 
-- run about tests: `npm run test:about:headed`
-- run av tests: `npm run test:av:headed`
-- run books tests: `npm run test:books:headed`
-- run collection tests: `npm run test:collection:headed`
-- run details tests: `npm run test:details:headed`
-- run home tests: `npm run test:home:headed`
-- run login tests: `npm run test:login:headed`
-- run music tests: `npm run test:music:headed`
-- run search tests: `npm run test:search:headed`
-- run profile tests: `npm run test:profile:headed`
+                `npm run test about trace`
+
+        - Test execution by browser:
+            - Tests will run in all browsers by default if you didn't specify the browser
+            - Accepted parameters: `chromium, firefox, webkit`
```

Comment:
BLOCKER: As with "trace" above, it is confusing to have both browser type and test type be represented by naked parameters. Otherwise, one might think that "firefox" is a test type like "donation". Browser types need their own command-line syntax, e.g. "--browser=chromium,firefox", or something equivalent.

Reply:
updated the code for consistency